plugin/plaintextmodel.cpp

```diff
@@ -24,16 +24,22 @@
 #include <QThread>
 #include <silicatheme.h>
 
+#include <unicode/ucsdet.h>
+
+#include <uchardet/uchardet.h>
+
 class PlainTextModel::FileData : public QSharedData
 {
 public:
     FileData(PlainTextModel *model)
         : fileName(model->m_file.fileName())
         , model(model)
+        , codec(nullptr)
     {
     }
 
     QString fileName;
+    QTextCodec * codec;
```

Comment:
Space after start and initialise to nullptr.

Reply:
You are right. I'll do it